xbmc/pictures/Picture.cpp

```diff
@@ -176,7 +180,7 @@ bool CPicture::ResizeTexture(const std::string &image, uint8_t *pixels, uint32_t
   // create a buffer large enough for the resulting image
   GetScale(width, height, dest_width, dest_height);
 
-  uint8_t *buffer = new uint8_t[dest_width * dest_height * sizeof(uint32_t)];
+  uint8_t *buffer = (uint8_t*) _aligned_malloc(dest_width * dest_height * sizeof(uint32_t), 32);
```

Comment:
this file already makes use of ffmpeg (sws_scale). use of av_malloc is welcome here